[17.0] [MIG] purchase_order_owner (#2347)
OCA/purchase-workflow: purchase_order_owner/readme/DESCRIPTION.md

This module adds Assign Owner (owner_id) field in purchase order, whose
value will be passed to the corresponding field of the incoming picking
upon order confirmation.

This module can be useful in a business situation where you purchase

> products in place of a client (you seprately issue an invoice to the

Reviewer: ```suggestion
products on behalf of a client (you separately issue an invoice to the
```
